### PR TITLE
Fix savings on iOS 12

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

File: src/helpers/isRainbowTextAvailable.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { UIManager } from 'react-native';
-import isNativeStackAvailable from '../helpers/isNativeStackAvailable';
 
-const useNativeButtonAvailable = () =>
-  isNativeStackAvailable && !!UIManager.getViewManagerConfig('RainbowText');
+const useRainbowTextAvailable = () =>
```
Comment (jinchung) [inline src/helpers/isRainbowTextAvailable.js:3]: can we name it to something not using 'use'? perhaps just `isRainbowTextAvaliable`

Comment (jinchung) [reply src/helpers/isRainbowTextAvailable.js:3]: also, is this something we can turn into a constant or does it need to be a function call?

Comment (osdnk) [reply src/helpers/isRainbowTextAvailable.js:3]: yeah, I don't really remember why it is like this

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**